html/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
+  </head>
+  <body>
+    <header>
+      <h1>Site description</h1>
+      <nav>
+        <a href="#">HTML</a> |
+        <a href="#">CSS</a> |
+        <a href="#">JavaScript</a> |
+        <a href="#">jQuery</a>
+      </nav>
+    </header>
+    <section>
+      <h1>WWF</h1>
+      <p>The World Wide Fund for Nature (WWF) is....</p>
+    </section>
+    <article>
+      <h1>What Does WWF Do?</h1>
+      <p>WWF's mission is to stop the degradation of our planet's natural environment,
+        and build a future in which humans live in harmony with nature.
+      </p>
+      <aside>
+        <h4>Epcot Center</h4>
+        <p>The Epcot Center is a theme park in Disney World, Florida.</p>
+      </aside> 
+    </article>
+    <main>
```

Comment:
main tag defines the main content in your document, for me it's like a wrapper for everything except footer, header, nav